src/AI/JakeMan.java

```diff
@@ -0,0 +1,770 @@
+package AI;
+
+import java.util.*;
+import java.util.Map.Entry;
+
+import AI.CommanderProductionInfo;
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import Engine.*;
+import Engine.UnitActionLifecycles.WaitLifecycle;
+import Terrain.*;
+import Units.*;
+
+
+public class JakeMan extends ModularAI
+{
+  private static class instantiator implements AIMaker
+  {
+    @Override
+    public AIController create(Army army)
+    {
+      return new JakeMan(army);
+    }
+
+    @Override
+    public String getName()
+    {
+      return "JakeMan";
+    }
+
+    @Override
+    public String getDescription()
+    {
+      return
+          "Wants your free dudes.\n" +
+          "High-level design by @Lost&Found#6348, ID 824112693123612692.";
```

Comment:
This... is not helpful.

Reply:
"Free dude" is a technical term, standing for "insufficiently-protected unit".
Simple AI, simple description.

Reply:
"Simple" is subjective, and relative. The "free dudes"/contested-square logic is more sophisticated than the other AIs have, and in any case, the descriptor "Free Dudes" isn't "simple", it is simply obtuse.
Imagine you have played this game before, but were not in any way involved in inventing, writing, or reviewing this source code. What does this description tell you? Absolutely nothing.

Reply:
The contested square logic is more sophisticated, yes, but the building algorithm is greatly simplified and a fair chunk of the code *around* the contested tile logic is also simplified.
"Free dudes" is only as obtuse as e.g. "linked list". It's technical jargon, but technical jargon I'd kind of expect our effective audience to understand.
If I'm an AWBWian, I would probably know what a "free dude" is. I guess "exploits openings in your defenses" isn't exactly a super great description either, though.

How about I add: "Built specifically for AWBW-meta PvP"?

Reply:
> technical jargon

Is "free dudes" really a common phrase that gets thrown around? At any rate, I'd still avoid technical jargon; at least anything that someone coming from the carts wouldn't know about (and they don't have much jargon anyway).

> How about I add: "Built specifically for AWBW-meta PvP"?

That may be overselling its capabilities. At least, I'm pretty sure artillery is pretty important in the AWBW meta. I'd probably just say it looks for favorable unit exchanges or something along that line.

Reply:
Probably also worth mentioning what it builds, since it has a pretty limited roster.